### PR TITLE
Remove *.toSource() and uneval()

### PR DESCRIPTION
**Pull request by teoli2003**
These methods, only supported in Firefox, got removed more than 2 years ago.

This is the counterpart of mdn/content#17982 and is part of of mdn/content#17962

### CODE REVIEW

File: javascript/builtins/Boolean.json [hunk 1]
```diff
@@ -91,93 +91,6 @@
...
-              "webview_android": "mirror"
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": false,
-              "deprecated": false
-            }
-          }
-        },
-        "toString": {
```
Comment (foolip) [inline javascript/builtins/Boolean.json:136]: Oops, this is `toString`, no `toSource`.

File: javascript/builtins/globals.json [hunk 2]
```diff
@@ -709,49 +709,6 @@
             "deprecated": true
           }
         }
-      },
-      "uneval": {
```
Comment (foolip) [inline javascript/builtins/globals.json:713]: Can you add this to the PR title/description, or split into a separate PR?

Comment (teoli2003) [reply javascript/builtins/globals.json:713]: I'm adding it to the title. They have been removed in the same Firefox bug, so it makes sense to keep their removal together here.

File: javascript/builtins/Boolean.json (hunk 1 above)
Comment (teoli2003) [reply javascript/builtins/Boolean.json:136]: Whoopsie, I readded `toString()`; sorry.

**Review by queengooborg — APPROVED** [review]
LGTM, thanks!
